README.md

```diff
@@ -30,7 +38,7 @@ curl -L https://github.com/redhat-developer/odo/raw/master/scripts/install.sh |
 </details>
 
 <details>
-<summary> :package: :apple: MacOS</summary>
+<summary> MacOS</summary>
```

Comment:
I like the emoji's :+1: I find that having them adds a few more graphics and centers the developers attention.. Especially to a page lacking graphics.

Reply:
I would encourage you to do away with these graphics for the following reasons:
- Figures or graphics should be used to clarify or complement information. 
- Lists, bulleted or numbered, are meant to attract the attention of the user, I am not sure we need graphics in addition here.
- A user who wants to install odo, would automatically click on the OS he uses. Once we move to Ascii doc, I would actually recommend using Definition list for this, which would make the names of the OS bold, automatically highlighting them for the users.
- We also need to keep our styles in sync with the conventions followed in the OC doc set, so far I haven't seen graphics used in this manner in the OC doc set.

Reply:
okay, that sounds good to me :+1: